pyEpiabm/pyEpiabm/intervention/case_isolation.py

```diff
@@ -43,3 +44,9 @@ def __call__(self, time):
                         if r < self.isolation_probability:
                             person.isolation_start_time = time + self.\
                                                           isolation_delay
+
+    def __turn_off__(self):
```

Comment:
This is not a system defined method so you should not use a dunder method (see [docs](https://docs.python.org/3/reference/lexical_analysis.html#reserved-classes-of-identifiers))